cmake/macros/macro_clear_feature.cmake

```diff
@@ -25,10 +25,7 @@
 #
 
 macro(CLEAR_FEATURE _feature)
```

Comment:
```suggestion
macro(clear_feature _feature)
```?

Reply:
Let me lowercase all of the macro names in a separate PR.

Reply:
Actually. Let me add it to this PR.